Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -271,6 +271,7 @@ - (void)setup {
     [self.calendarWrapperView addSubview:self.calendarView];
 
     self.calendarWeekdayView = [[FSCalendarWeekdayView alloc] initWithFrame:CGRectZero];
+    self.calendarWeekdayView.accessibilityElementsHidden = YES;
```

Comment:
This will make sure we don't go through the 
"Mon" "Tue" "Wed" "Thu" "Fri" "Sat" "Sun" labels at the top of the calendar. They add no value for screen reader users.

The change above, to make the date format longer will make it much clearer what day a date is on.